### PR TITLE
Split out calendar invite lists

### LINKED ISSUE #1414: Public calendaring issue/improvement
We are having some problems with the Google calendar syncing (for those who have joined [opentelemetry-calendar@googlegroups.com](mailto:opentelemetry-calendar@googlegroups.com)). Likely because I updated too many meetings while giving all of the Zoom meetings separate ids/names, and hit some Google calendar spamming limits (by the way, sorry for the spam 😅).

I'd like to propose a change (originally raised by @tylerbenson), and create separate "invitee" google groups per meeting so that we aren't spamming so many updates for every meeting (there are currently >150 people in opentelemetry-calendar@googlegroups.com).

If we do this, I'd like to remove opentelemetry-calendar@googlegroups.com from the meetings, and email that list letting them know about the new per-meeting google groups they can join instead.

If we still need a list for all meetings, we can create that separately (e.g. opentelemetry-calendar-all@googlegroups.com) and hopefully it should be much smaller number of people who want to be in that group.

(and yes, I'm volunteering to make these changes and document them in this repo, I've lost too much time on the calendar in the last days to not hopefully see this through to a better place 😅)

### PR DESCRIPTION
**Pull request by trask**
Resolves #1414

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ JavaScript: SDK|Every Wednesday at 09:00 PT| [opentelemetry-calendar-js@googlegr
...
-Rust: SDK|N/A| [opentelemetry-calendar-rust@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-rust)                             | [Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)                             |                                                                                                 |
+Rust: SDK|Every Tuesday at 09:00 PT| [opentelemetry-calendar-rust@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-rust)                             | [Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)                             |                                                                                                 |
```
Comment (tylerbenson) [inline README.md:151]: Is it worth adding AM/PM to all the times?

Comment (trask) [reply README.md:151]: they are currently all in 24 hour time

**Review by arminru — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -122,44 +122,43 @@ SIG-specific GitHub discussions.
...
+Ruby: SDK|Every Tuesday at 09:00 PT| [opentelemetry-calendar-ruby@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-ruby)                             | [Google Doc](https://docs.google.com/document/d/1D15bO8o340sQm2CVZiukEJuCO_XMMHKPuTznoEhyFqE/edit)                        | [#otel-ruby](https://cloud-native.slack.com/archives/C01NWKKMKMY) and [GitHub Discussions](https://github.com/open-telemetry/opentelemetry-ruby/discussions) |
+Rust: SDK|Every Tuesday at 09:00 PT| [opentelemetry-calendar-rust@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-rust)                             | [Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)                             |                                                                                                 |
```
Comment (arminru) [inline README.md:151]: You could add their Slack channel as well (see @hdost's PR #1418):
```suggestion
Rust: SDK|Every Tuesday at 09:00 PT| [opentelemetry-calendar-rust@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-rust)                             | [Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)                             |[#otel-rust](https://cloud-native.slack.com/archives/C03GDP0H023) and [Gitter](https://gitter.im/open-telemetry/opentelemetry-rust)|
```

Comment (hdost) [reply README.md:151]: @trask if you want to cherry pick my change that would be preferable.

File: README.md [hunk 3]
```diff
@@ -122,44 +122,43 @@ SIG-specific GitHub discussions.
...
+Semantic Conventions: General|Every other Monday at 10:00 PT| [opentelemetry-calendar-semconv-general@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-semconv-general)       | [Google Doc](https://docs.google.com/document/d/10xG7DNKWRhxNmFGt3yYd3980a9uwS8lMl2LvQL3VNK8)      | [#otel-semantic-conventions-wg](https://cloud-native.slack.com/archives/C041APFBYQP) 
+Semantic Conventions: Messaging|Every Thursday at 8:00 PT| [opentelemetry-calendar-semconv-messaging@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-semconv-messaging)   | [Google Doc](https://docs.google.com/document/d/1dWHhyXnfVife-cQ2DW5-d5Ldp1Lq8Rre2UsHpyo8cEE/)     | [#otel-semantic-conventions-wg](https://cloud-native.slack.com/archives/C041APFBYQP)
```
[the remark this answers is not in the document]
Comment (trask) [reply README.md:132]: ya, I think that's a good idea, will do 👍

File: assets.md [hunk 4]
```diff
@@ -118,6 +118,12 @@ Link: https://splain.io/
 - Used to manage the OpenTelemetry community calendar and Zoom
 - Owned by the governance committee
 
+### Project Google account: opentelemetry.calendar.owner@gmail.com
+
+- Used to own calendar-related things (a separate account account is used just
+  for this since it gets a lot of email calendar invites).
+- Owned by the governance committee
```
[the remark this answers is not in the document]
Comment (trask) [reply assets.md:125]: (no 2FA on the account)

File: README.md (hunk 2 above)
Comment (trask) [reply README.md]: @hdost I left a couple comments on your PR to clarify

**Review by jsuereth — APPROVED** [review]

**Review by dyladan — APPROVED** [review]

**Comment by trask** [comment]
putting this on hold until we look into #1440, I'd rather only break everything one (more) time

**Comment by trask** [comment]
Replaced by #1788

**Pull request closed without merging**